spec/models/device_group_spec.rb

```diff
@@ -1,5 +1,26 @@
 require 'rails_helper'
 
 RSpec.describe DeviceGroup, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  subject {
```

Comment:
create factory for DeviceGroup + test for valid factory, use let http://www.betterspecs.org/#let